src/uct/ib/base/ib_iface.c

```diff
@@ -1244,7 +1244,7 @@ UCS_CLASS_INIT_FUNC(uct_ib_iface_t, uct_ib_iface_ops_t *ops, uct_md_h md,
     self->config.rx_max_batch       = ucs_min(config->rx.max_batch,
                                               config->rx.queue_len / 4);
     self->config.port_num           = port_num;
-    self->config.sl                 = UCT_IB_SL_INVALID;
+    self->config.sl                 = UCT_IB_SL_NUM;
```

Comment:
add comment /* initialize to invalid value */

Reply:
done